community/contribution_guide/release.md

```diff
@@ -385,86 +403,75 @@ Checklist for reference:
 
 [ ] No compiled archives bundled in source archive.
 
-More detail checklist please refer:
-https://cwiki.apache.org/confluence/display/INCUBATOR/Incubator+Release+Checklist
-
 
---
+— 
 
 Best Regards
-Chao Tian
+Naijie Liu
+
 ```
 
 #### Voting close
 ```bash
-[VOTE] Release Apache SeaTunnel 2.3.3
-
-Hi SeaTunnel Community,
-
-Thanks, everyone, I will close this vote thread and the results will be tallied.
-
-Best wishes!
-Chao Tian
-#### 归票
-[RESULT] [VOTE] Release Apache SeaTunnel 2.3.3
+[RESULT] [VOTE] Release Apache SeaTunnel 2.3.8
```

Comment:
Lets start with release condidate 0
Change "[VOTE] Release Apache SeaTunnel 2.3.8" to "[VOTE] Release Apache SeaTunnel 2.3.8 condidate 0"